packages/@glimmer/program/lib/program.ts

```diff
@@ -194,17 +194,7 @@ export class ProgramImpl implements Program {
 }
 
 function slice(arr: Int32Array, start: number, end: number): Int32Array {
-  if (arr.slice !== undefined) {
```

Comment:
No supported engines are missing `slice` anymore. Further, this whole function only existed for
eager mode, which doesn't exist anymore.